src/chapters/protocol.xml

```diff
@@ -80,6 +80,22 @@
             it supports Ogg Vorbis audio, but no video, and can accept both PNG and JPEG images. It
             wants to connect to localhost at port 5900, and is leaving the three other parameters
             blank.</para>
+        <para>Version 1.1.0 of Guacamole adds a couple of other features to the handshake phase.  First,
+            the handshake can now be done with most of the handshake instructions sent in arbitrary order.
+            Beginning with this version, guacd will detect and parse out each of the options regardless
+            of the order they are sent in, looking for the connect instruction as the indication
+            that the client has finished sending options.  In addition to ordering, this means
+            that certain instructions can be omitted without preventing the connection from
+            continuing.
+        </para>
+        <para>Also introduced in version 1.1.0 is the ability to pass the client timezone to
```

Comment:
Now that the handshake is evolving, it may be easier to just list out the handshake instructions as a table. I don't really have a problem with prose, and I generally like how you've phrased things here, but I do think things should be described in a more absolute way, not layers of relative additions.

For example, these changes currently describe things more-or-less like:

* The protocol handshake requires you to provide `size`, `audio`, `video`, and `image`.
* But actually, never mind. That's the old protocol. Those are optional now and may occur in any order.
* Wait! There's also a new `timezone` instruction you can use.

I'd lean more towards:

* The protocol handshake consists of any of the following instructions, which are each optional and may occur in any order: (instructions....)
* **NOTE:** Older releases supported a smaller subset of these instructions and required handshake instructions to be in a specific order. See the archived versions of the manual for the protocol documentation relevant to older releases.

Reply:
Sounds good - will try some different language.

Reply:
Okay, have taken a stab at shamelessly copying a table from elsewhere in the manual and substituting in the handshake instructions.  Let me know how it looks - currently compiling it to make sure everything formats well, but I believe I at least got valid XML.